set.mm

```diff
@@ -39637,6 +39637,12 @@ among classes ( ~ eq0 , as opposed to the weaker uniqueness among sets,
     ( weq wn cab cv wcel wa crab pm3.24 equid 2th con1bii dfnul2 df-rab 3eqtr4i
     c0 abbii ) AACZDZAEAFBGZUADZHZAEQUBABITUCAUCSUCDSUAJAKLMRANUBABOP $.
 
+  $( Alternate definition of the empty class/set.  (Contributed by BJ,
+     30-Nov-2019.) $)
```

Comment:
Probably credit is due to you, here. Something like `Avoid ~ ax-8 , ~ df-clel .  (Revised by GG, ...`